tendrl/node_agent/manager/manager.py

```diff
@@ -227,6 +228,34 @@ def on_pull(self, raw_data):
                     node_id=raw_data["node_id"],
                 )
             )
+        if "disks" in raw_data:
+            try:
+                self.etcd_client.delete(
+                    ("nodes/%s/Disks") % raw_data["node_id"], recursive=True)
+            except Exception as ex:
```

Comment:
Figure our the broadest exception thrown by python-etcd when a given key is not found in etcd and catch it